src/scss/3_components/content/_icons-messages.scss

```diff
@@ -186,4 +186,5 @@ ul.info-notice {
 
 .info-notice {
   overflow: hidden;
+  border-color: #004695;
```

Comment:
Just a thought - as we are using Sass, wouldn't it be a good idea to define colours like this somewhere centrally, rather than hard coding RGB values ?

Reply:
Yes, good point, I think this will be the same throughout all UNITY sites but I have now defined the colour as a variable incase it does need to be changed on a per site basis.